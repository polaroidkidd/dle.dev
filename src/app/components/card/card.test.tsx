import React from 'react';
import renderer from 'react-test-renderer';
import Card from './card';


test('Render Card Component', () => {
  const tree = renderer.create(
    <Card />).toJSON();
  expect(tree).toMatchSnapshot();
});
