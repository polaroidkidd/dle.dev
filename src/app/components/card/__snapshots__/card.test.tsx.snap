// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Card Component 1`] = `
<div
  className="card"
>
  <h2>
    This is a card
  </h2>
</div>
`;
